docs/UserGuide.md

```diff
@@ -383,7 +383,7 @@ Fields supported in `find`:
 - Class Date `dt/`
 - Tag `t/`
 
-<div markdown="span" class="alert alert-danger">❗ **Caution:** Only one field can be searched at once.
+<div markdown="span" class="alert alert-danger">❗ **Caution:** Only one prefix should be used in a find command at a time.
```

Comment:
As I raised in the issue #307 , I think we need to further clarify about the sentence here.

Reply:
The issue #307 is not planned to be implemented.